### DISCUSSION TITLE
Next.js 使用 Hono 接管 API | 愧怍

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#320] [General]
# Next.js 使用 Hono 接管 API | 愧怍

愧怍的个人博客

https://kuizuo.cn/blog/nextjs-with-hono/

**Comment by NokiOvO** [comment]
大佬！！！解决了一个初学者的痛点！！！！实在太感谢了啊啊啊啊啊啊啊啊啊啊

**Comment by Nick742037091** [comment]
为啥不直接用Server Action呢

**Reply by kuizuo** [reply]
使用 server action 就无法暴露接口了，而 hono 本质上还是next.js 的 page router 那一套 API 设计

**Reply by Nick742037091** [reply]
确实，Server Action不方便定位问题

**Comment by AaronConlon** [comment]
谢谢分享

**Comment by chenbj5515** [comment]
一般什么样的项目会用NextJS的同时还需要暴露很多接口？给移动端用吗？
